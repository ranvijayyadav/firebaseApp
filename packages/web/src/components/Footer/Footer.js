
import React from 'react';
//import { Container, Row, Column, Heading } from 'rebass';
import './Footer.css';
const Footer = () => {
  return (
    <footer className="page-footer font-small blue">

      <div className="footer footer-copyright text-center py-3">Â© 2020 Copyright:
        footer
    </div>

    </footer>
  );
};

export default Footer;

