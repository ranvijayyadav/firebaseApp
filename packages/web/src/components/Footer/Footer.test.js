import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

let wrapped = shallow(<Footer />);
describe('Footer', () => {
    it("Should render Footer properly", () => {
        expect(wrapped).toMatchSnapshot();
    });
})