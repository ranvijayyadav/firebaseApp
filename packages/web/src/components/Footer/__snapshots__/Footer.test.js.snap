// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Should render Footer properly 1`] = `
<footer
  className="page-footer font-small blue"
>
  <div
    className="footer footer-copyright text-center py-3"
  >
    Â© 2020 Copyright: footer
  </div>
</footer>
`;
